docs/docs/features/xmp-sidecars.md

```diff
@@ -1,13 +1,73 @@
+![XMP sidecars](./img/xmp-sidecars.webp)
 
-There are 2 administrator jobs associated with sidecar files: `SYNC` and `DISCOVER`. The sync job will re-scan all media with existing sidecar files and queue them for a metadata refresh. This is a great use case when third-party applications are used to modify the metadata of media. The discover job will attempt to scan the filesystem for new sidecar files for all media that does not currently have a sidecar file associated with it.
+## 1 · What Immich Reads
 
-<img src={require('./img/sidecar-jobs.webp').default} title='Sidecar Administrator Jobs' />
+Only a few XMP fields are saved into the database:
+
+| Stored in DB       | XMP Field                         |
+|--------------------|-----------------------------------|
+| **Title**          | `dc:title`                        |
+| **Description**    | `dc:description`                  |
+| **DateTimeOriginal** | `exif:DateTimeOriginal`        |
+| **Rating**         | `xmp:Rating`                      |
+| **Tags**           | `digiKam:TagsList`, `dc:subject`  |
+
+All other fields (e.g. `Creator`, `Source`, IPTC, Lightroom edits) remain in the `.xmp` file and are **not searchable** in Immich.
+
+---
+
+## 2 · File Naming Rules
+
+A sidecar must share the base name of the media file:
+
+- ✅ `IMG_0001.jpg.xmp` ← preferred
+- ✅ `IMG_0001.xmp` ← fallback
+- ❌ `myphoto_meta.xmp` ← not recognized
+
+If both `.jpg.xmp` and `.xmp` are present, Immich uses the **`.jpg.xmp`** file.
+
+---
+
+## 3 · How Sidecars Are Handled
+
+### A · Imported via CLI
+
+When uploading media via the CLI:
+
+1. **Detect** – Immich looks for a `.xmp` file next to each media file
+2. **Copy** – Both media and sidecar are copied into Immich's internal storage. The `.xmp` file is renamed to follow the same naming pattern as the media based on storage template eg. `upload/library/<user>/YYYY/YYYY-MM-DD/IMG_0001.jpg` and corresponding `upload/library/<user>/YYYY/YYYY-MM-DD/IMG_0001.jpg.xmp`
+3. **Extract** – Metadata (title, description, date, rating, tags) is saved to the database
+4. **Write-back** – If you later change tags, rating, etc. in the web UI, Immich updates **both** the database *and* the sidecar file
+
+### B · External Library (Mounted Folder)
+
+For read-only or externally mounted libraries:
+
+1. **Detect** – The `DISCOVER` job links `.xmp` files that sit next to media (they are **not** copied)
+2. **Extract** – Metadata is imported into the database
+3. **Write-back** – If Immich has **write access**, sidecar files are updated with your changes  
+- If the mount is **read-only**, changes only affect the database
```

Comment:
We don’t support editing fields for read only external libraries. If the sidecar can’t be written, the DB will not be written either.